WebRoot/js/ajax/dwt/widgets/DwtIframe.js

```diff
@@ -261,7 +261,7 @@ DwtIframe.prototype._createFrame = function(html) {
 		if(self._onLoadHandler){
 			tmp[i++] = " onload='" + self._onLoadHandler + "'";
 		}
-		tmp[i++] = " src='javascript:\"\";' ></iframe>";
```

Comment:
instead of removing src, can we set it to about:blank? as per https://www.w3.org/TR/2011/WD-html5-20110525/fetching-resources.html#about:blank, content will be immediately available

Reply:
Fixed. and working fine as far as i check.